### PR TITLE
Webpack and Dependencies initialization

### PR DESCRIPTION
**Pull request by phillipkb**
## **Types of Changes**
- [ ] Bugfix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Refactor (change which changes the codebase without affecting its external behavior)
- [ ] Non-breaking change (fix or feature that would causes existing functionality to work as expected)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## **Purpose**
initialize webpack and dependencies

## **Approach**
- install dependencies
- configure webpack with dev server 

## **How to Test**
- npm run build // should create dist folder with bundled files
- npm start // should start app in production mode on port 3000
- npm run dev // should open tab on 8000, proxied to 3000, with HMR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Getting Started</title>
```
Comment (annieshinn) [inline index.html:5]: Do we want to update this with this PR?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "reinforcement",
+  "version": "1.0.0",
+  "description": "reinforcement project repo",
```
Comment (annieshinn) [inline package.json:4]: Let's make this more relevant to the project at hand (InterView or interView or inter-view or whatever it's official name will be!)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  "main": "Main.tsx",
+  "scripts": {
+    "test": "",
+    "build": "cross-env NODE_ENV=production webpack",
+    "start": "ts-node server/server.ts",
```
Comment (annieshinn) [inline package.json:9]: Let's declare a NODE_ENV with this command as well.

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+};
+
+//nodemon file contents
+
+// {
+//   "watch": ["src"],
+//   "ext": "ts,json",
+//   "ignore": ["src/**/*.spec.ts"],
+//   "exec": "ts-node ./src/index.ts"      
+// }
```
Comment (annieshinn) [inline webpack.config.js:53]: Clean up, please. :D

**Review by annieshinn — APPROVED** [review]
Hey Phil!

Thanks for this setup. I left some comments throughout, and noted we will shorten the .gitignore file.

Thanks!

**Review by Daniel-P3 — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (annieshinn) [reply package.json:9]: Oops, I meant the "start" script.

File: index.html (hunk 1 above)
Comment (phillipkb) [reply index.html:5]: No, changing it to Inter-views, thank you.

File: package.json (hunk 2 above)
Comment (phillipkb) [reply package.json:4]: Thank you, updated

File: package.json (hunk 3 above)
Comment (phillipkb) [reply package.json:9]: Thank you, mode=production added

File: webpack.config.js (hunk 4 above)
Comment (phillipkb) [reply webpack.config.js:53]: What would we do without you

**Review by annieshinn — APPROVED** [review]
I've tested the app again, both in production and development mode.  Everything seems as described. Let's get it!! 🚀 

Thanks for your hard work and for handling all comments, Phil!

**Review by bielchristo — APPROVED** [review]

**Pull request merged**